commit f67feea0934f16ec256077b8d108284e4896d950
Merge: 9f7f735 6c050b8
Author: ErikZalm <erik@vdzalm.eu>
Date:   Mon Aug 20 14:34:32 2012 -0700

    Merge pull request #217 from Justblair/Marlin_v1
    
    Fixed broken Language file and pre-heat ABS and PLA options

diff --git a/Marlin/language.h b/Marlin/language.h
index 12461a6..99c6751 100644
--- a/Marlin/language.h
+++ b/Marlin/language.h
@@ -37,6 +37,7 @@
 	#define MSG_SET_ORIGIN " Set Origin"
 	#define MSG_COOLDOWN " Cooldown"
 	#define MSG_EXTRUDE " Extrude"
+	#define MSG_RETRACT " Extract"
 	#define MSG_PREHEAT_PLA " Preheat PLA"
 	#define MSG_PREHEAT_ABS " Preheat ABS"
 	#define MSG_MOVE_AXIS " Move Axis      \x7E"
@@ -87,6 +88,8 @@
 	#define MSG_CONTROL_ARROW " Control \x7E"
 	#define MSG_RETRACT_ARROW " Control \x7E"
 	#define MSG_TUNE " Tune    \x7E"
+	#define MSG_PAUSE_PRINT " Pause Print \x7E"
+	#define MSG_RESUME_PRINT " Resume Print \x7E"
 	#define MSG_STOP_PRINT " Stop Print   \x7E"
 	#define MSG_CARD_MENU " Card Menu    \x7E"
 	#define MSG_NO_CARD " No Card"
diff --git a/Marlin/ultralcd.pde b/Marlin/ultralcd.pde
index 9be357c..98d9f4b 100644
--- a/Marlin/ultralcd.pde
+++ b/Marlin/ultralcd.pde
@@ -530,14 +530,16 @@ void MainMenu::showPrepare()
     case ItemP_preheat_pla:
       MENUITEM(  lcdprintPGM(MSG_PREHEAT_PLA)  ,  BLOCK;setTargetHotend0(PLA_PREHEAT_HOTEND_TEMP);setTargetBed(PLA_PREHEAT_HPB_TEMP);
       #if FAN_PIN > -1
-        analogWrite(FAN_PIN, PLA_PREHEAT_FAN_SPEED);
+		FanSpeed=PLA_PREHEAT_FAN_SPEED;
+        analogWrite(FAN_PIN,  FanSpeed);
       #endif
       beepshort(); );
       break;
     case ItemP_preheat_abs:
       MENUITEM(  lcdprintPGM(MSG_PREHEAT_ABS)  ,  BLOCK;setTargetHotend0(ABS_PREHEAT_HOTEND_TEMP);setTargetBed(ABS_PREHEAT_HPB_TEMP); 
       #if FAN_PIN > -1
-        analogWrite(FAN_PIN, ABS_PREHEAT_FAN_SPEED);
+	  	FanSpeed=ABS_PREHEAT_FAN_SPEED;
+        analogWrite(FAN_PIN,  FanSpeed);
       #endif
       beepshort(); );
       break;
